ajax/libs/filesize/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "filesize",
+  "version": "3.5.6",
+  "filename": "filesize.min.js",
+  "description": "JavaScript library to generate a human readable String describing the file size",
+  "homepage": "https://filesizejs.com/",
+  "keywords": [
+    "file",
+    "filesize",
+    "size",
+    "readable",
+    "file system",
+    "bytes",
+    "diff"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/avoidwork/filesize.js"
```

Comment:
Maybe we can follow the format as the same as upstream. ref: https://github.com/avoidwork/filesize.js/blob/master/package.json#L9
`"url": "git://github.com/avoidwork/filesize.js.git"`